engine/rowset/src/main/java/io/deephaven/engine/rowset/impl/WritableRowSetImpl.java

```diff
@@ -77,6 +77,14 @@ protected void preMutationHook() {}
 
     protected void postMutationHook() {}
 
+    @Override
```

Comment:
You can document the implementation details in the implementation.
```suggestion
    /**
     * 
     * Reset this WritableRowSetImpl to match another RowSet by updating the {@code innerSet}. This will internally
     *  assign {@code this.innerSet} to a copy-on-write reference to {@code ((WritableRowSetImpl) other).innerSet}. 
     * The two sets will diverge when either is modified. To maintain continuity between the sets, this function should
     * be called each UGP cycle.
     *
     * @param other The RowSet to reset to
     */
    @Override
```